<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskDilationWidget</class>
 <widget class="QWidget" name="MaskDilationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="preview_checkbox">
     <property name="toolTip">
      <string>Enable real-time preview of mask dilation changes</string>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mode_layout">
     <item>
      <widget class="QLabel" name="mode_label">
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="grow_radio">
       <property name="toolTip">
        <string>Grow (dilate) the mask to make it larger</string>
       </property>
       <property name="text">
        <string>Grow</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="shrink_radio">
       <property name="toolTip">
        <string>Shrink (erode) the mask to make it smaller</string>
       </property>
       <property name="text">
        <string>Shrink</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="size_layout">
     <item>
      <widget class="QLabel" name="size_label">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="size_spinbox">
       <property name="toolTip">
        <string>Size of the dilation/erosion operation (1-100 pixels)</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="apply_button">
       <property name="toolTip">
        <string>Apply the dilation changes permanently to the mask data</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 